src/pages/home/songs/SongGrid.tsx

```diff
@@ -0,0 +1,104 @@
+import { Box, Grid, Typography } from "@mui/material";
+import addSong from "assets/images/add-song.png";
+import { useAppSelector } from "common";
+import { FilledButton, PinkTypography } from "components";
+import { History } from "history";
+import { selectSongs } from "modules/song";
+import { Dispatch, SetStateAction } from "react";
+import AddSongCard from "./AddSongCard";
+import Song from "./Song";
+
+export interface SongsProps {
+  history: History;
+  setOpenPopup: Dispatch<SetStateAction<boolean>>;
+}
+
+const Songs = ({ history, setOpenPopup }: SongsProps) => {
+    const songs = useAppSelector(selectSongs);
+  if (songs[0] === undefined) {
+    return (
+      <div style={ { height: "100%", width: "100%" } }>
```

Comment:
`songs` will actually be an object, so `songs[0]` will only check if a song with an ID of 0 exists. You'd need to do something like 

```
const isSongsEmpty = Object.values(songs).length === 0;
```

Also, in order to clean up the component, this coud be moved to a separate component and imported. E.g.

```
import StartParty from "./StartParty"

...
if (isSongsEmpty) {
  return <StartParty />;
}

...
```